source/includes/example-connect.rst

```diff
@@ -0,0 +1,54 @@
+Gather Connection Information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The source cluster, ``cluster0``, is hosted on the following servers
+and ports:
+
+- clusterOne01.fancyCorp.com:20020
+- clusterOne02.fancyCorp.com:20020
+- clusterOne03.fancyCorp.com:20020
+
+The destination cluster, ``cluster1``, is hosted on the following
+servers and ports:
+
+- clusterTwo01.fancyCorp.com:20020
+- clusterTwo02.fancyCorp.com:20020
+- clusterTwo03.fancyCorp.com:20020
+
+There is an administrative user, ``clusterAdmin`` configured on each
+cluster with password, ``superSecret``.
+
+Connect the Source and Destination Clusters with ``mongosync``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The generic connection string format is: 
+
+.. code-block:: shell
+
+   mongodb://<user>:<password>@<ip-address>:<port>,<ip-address>:<port>,<ip-address>:<port>
+
+Use the connection information you gathered to create the connection
+strings for ``cluster0`` and ``cluster1``:
+
+.. code-block:: shell
```

Comment:
[rendering note]

Just a heads up, it looks like using :copyable:false disables horizontal scrolling on the rendered pages.
<img width="847" alt="Screen Shot 2022-06-02 at 4 01 21 PM" src="https://user-images.githubusercontent.com/73852296/171728009-461a8788-3e38-4988-9214-0cdfae67e76b.png">